packages/solid/store/src/store.ts

```diff
@@ -47,13 +47,13 @@ export function isWrappable(obj: any) {
   );
 }
 
-export function unwrap<T extends StoreNode>(item: any, set = new Set()): T {
+export function unwrap<T extends StoreNode>(item: T, set = new Set()): T {
```

Comment:
The return type should be `DeepMutable<T>`, otherwise it will remain readonly despite being unwrapped.